Podcast/UserDetailSectionHeaderView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ProfileSectionHeaderView.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 11/16/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+enum UserDetailSectionHeaderType: String {
+    case subscriptions = "Public Subscriptions"
```

Comment:
Can we make it just say "Subscriptions" for now and not "Public Subscriptions" since we don't have private subscriptions at this time.

Reply:
Sure